tools/transformer/src/atlasTransformations.js

```diff
@@ -67,3 +71,43 @@ module.exports = function runTransformations(openapi) {
 
   return openapi;
 };
+
+// Workarounds for issues in the search API
+function searchAPIIssuesTransformation(openapi) {
+  // API is overly complex in this case and provides no value as typed interface
+  if (openapi.components.schemas.SearchIndexResponse) {
+    const responseParent = openapi.components.schemas.SearchIndexResponse;
+    if (
+      responseParent.properties &&
+      responseParent.properties.latestDefinition
```

Comment:
do we have modern js available?
```suggestion
      responseParent?.properties?.latestDefinition
```

Reply:
Sadly we cannot make that assumption - due to internal system CI/CD. We had issues with `?` not being recognized.